MIDAS/src/hardware/Voltage.cpp

```diff
@@ -1,17 +1,19 @@
 #include "sensors.h"
+#include <hal.h>
```

Comment:
If this include is here just to get `analogRead`, it's probably better to just include `Arduino.h` instead.